scss_lint.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 1.9.3'
 
-  s.add_dependency 'rake', '~> 10.0'
+  s.add_dependency 'rake', '>= 0.9', '~> 10.0'
```

Comment:
Should this be:

```ruby
s.add_dependency 'rake', '>= 0.9', '< 11'
```

My understanding is that all constraints must be satisfied, so `~> 10.0` will sill require you to have rake >= 10.0.